ckb-bin/src/lib.rs

```diff
@@ -56,8 +61,64 @@ pub fn run_app(version: Version) -> Result<(), ExitCode> {
     let (cmd, matches) = app_matches
         .subcommand()
         .expect("SubcommandRequiredElseHelp");
-    let is_silent_logging = is_silent_logging(cmd);
 
+    if run_deamon(cmd, matches) {
+        run_app_in_daemon(version, bin_name, cmd, matches)
+    } else {
+        debug!("ckb version: {}", version);
+        run_app_inner(version, bin_name, cmd, matches)
+    }
+}
+
+fn run_app_in_daemon(
+    version: Version,
+    bin_name: String,
+    cmd: &str,
+    matches: &ArgMatches,
+) -> Result<(), ExitCode> {
+    eprintln!("starting CKB in daemon mode ...");
+    eprintln!("check status : `{}`", "ckb daemon --check".green());
+    eprintln!("stop daemon  : `{}`", "ckb daemon --stop".yellow());
+
+    assert!(matches!(cmd, cli::CMD_RUN));
+    let root_dir = Setup::root_dir_from_matches(matches)?;
+    let daemon_dir = root_dir.join("data/daemon");
```

Comment:
We usually create the pid file in the tmp directory
```suggestion
    let daemon_dir = root_dir.join("tmp/daemon");
```

Reply:
@zhangsoledad suggestted add a new `daemon` sub-dirctory, 
which we can also store other things from daemon in future, and a fixed path may be better?
I think both are ok, if we don't cleanup tmp.
And we also won't create `daemon` sub-directory if user don't start ckb as daemon service.